packages/app/src/hooks/useZkVerify.ts

```diff
@@ -72,7 +73,7 @@ export function useZkVerify() {
         throw new Error(`Transaction failed: ${(error as Error).message}`);
       }
 
-      if (transactionInfo && transactionInfo.attestationId) {
+      if (transactionInfo && transactionInfo.statement && transactionInfo.aggregationId >= 0) {
```

Comment:
There is a type error and the build is failing. Logs:
```

07:48:25.456 | ▲  Type error: Property 'attestationId' does not exist on type 'VerifyTransactionInfo'.
-- | --
07:48:25.456 | ▲
07:48:25.456 | ▲    87 \|     const transactionInfo = await onVerifyProof(proofData, puzzle, vkey);
07:48:25.456 | ▲    88 \|     if (transactionInfo) {
07:48:25.457 | ▲  > 89 \|       message.success(`Verified Successfully on zkVerify - AttestationId: ${transactionInfo.attestationId}`);
07:48:25.457 | ▲       \|                                                                                             ^
07:48:25.457 | ▲    90 \|       downloadTransactionInfo(transactionInfo);
07:48:25.457 | ▲    91 \|     }
07:48:25.457 | ▲    92 \|   };


```

Reply:
Doh missed one.  Thanks should be replaced with aggregationId / domainId